src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotChangeEventSource.java

```diff
@@ -34,33 +50,61 @@
 
 public class YugabyteDBSnapshotChangeEventSource extends AbstractSnapshotChangeEventSource<YBPartition, YugabyteDBOffsetContext> {
     private static final Logger LOGGER = LoggerFactory.getLogger(YugabyteDBSnapshotChangeEventSource.class);
-    private final YugabyteDBConnectorConfig connectorConfig;
+    private final YugabyteDBConnectorConfig ybConnectorConfig;
     private final YugabyteDBSchema schema;
     private final SnapshotProgressListener snapshotProgressListener;
     private final YugabyteDBTaskContext taskContext;
     private final EventDispatcher<YBPartition,TableId> dispatcher;
     protected final Clock clock;
     private final Snapshotter snapshotter;
+    private final YugabyteDBConnection connection;
+
+    private final AsyncYBClient asyncClient;
+    private final YBClient syncClient;
+
+    private OpId lastCompletelyProcessedLsn;
+
+    private YugabyteDBTypeRegistry yugabyteDbTypeRegistry;
+
+    private final Metronome retryMetronome;
 
     public YugabyteDBSnapshotChangeEventSource(YugabyteDBConnectorConfig connectorConfig,
                                                YugabyteDBTaskContext taskContext,
-                                               Snapshotter snapshotter,
+                                               Snapshotter snapshotter, YugabyteDBConnection connection,
                                                YugabyteDBSchema schema, YugabyteDBEventDispatcher<TableId> dispatcher, Clock clock,
                                                SnapshotProgressListener snapshotProgressListener) {
         super(connectorConfig, snapshotProgressListener);
-        this.connectorConfig = connectorConfig;
+        this.ybConnectorConfig = connectorConfig;
```

Comment:
Why is the name change required?

Reply:
There was a `connectorConfig` variable in the parent class as well which was getting initialized in the super constructor. I was just playing around with the name change, have reverted the name change now.